os/kernel/init/os_bringup.c

```diff
@@ -286,7 +286,6 @@ static inline void os_do_appstart(void)
 		svdbg("Failed to create Task Manager\n");
 	}
 #endif
-
```

Comment:
unnecessary changes.

Reply:
Thanks for spotting. Removed the offending line.